juniper_official/Chassis/netsvc.rule

```diff
@@ -0,0 +1,37 @@
+iceberg {
+    topic chassis.networkservices {
+        rule netsvc-rule {
+            sensor netsvc-sensor {
+                iAgent {
+                    file netsvc.yml;
+                    table NetSvcTable;
+                    frequency 60s;
+                }
+            }
+            trigger netsvc-sync {
+                term synched {
+                    when {
+                        matches-with "$netsvc_sync" True;
+                    }
+                    then {
+                        status {
+                            color green;
+                        }
+                    }
+                }
+                term else {
+                    when {
+                        matches-with "$netsvc_sync" False;
+                        exists "$proposed_action";
```

Comment:
If netsvc_sync is false and there is no proposed action, don't we still flag it as red and say proposed action could not be compiled or something?

Reply:
proposed action is always a string because we can't reference variables in rules that don’t exist in influx (all referenced fields must be populated) or kapacitor silently does nothing and you see 'no data' in gui.

Currently, if no action is required the string is ''no action required"